tests/cupy_tests/lib_tests/test_polynomial.py

```diff
@@ -380,46 +387,52 @@ def test_poly1d_ne2(self, xp, dtype):
 
 @testing.gpu
 @testing.parameterize(*testing.product({
-    'fname': ['polyadd', 'polysub'],
+    'fname': ['polyadd', 'polysub', 'polymul'],
     'shape1': [(), (0,), (3,), (5,)],
-    'shape2': [(), (0,), (3,), (5,)]
+    'shape2': [(), (0,), (3,), (5,)],
 }))
 class TestPolyArithmeticShapeCombination(unittest.TestCase):
 
     @testing.for_all_dtypes(no_bool=True)
-    @testing.numpy_cupy_array_equal()
+    @testing.numpy_cupy_allclose(rtol=1e-5)
     def test_polyroutine(self, xp, dtype):
         func = getattr(xp, self.fname)
         a = testing.shaped_arange(self.shape1, xp, dtype)
         b = testing.shaped_arange(self.shape2, xp, dtype)
-        with cupyx.allow_synchronize(False):
-            return func(a, b)
+        return func(a, b)
 
 
 @testing.gpu
 @testing.parameterize(*testing.product({
-    'fname': ['polyadd', 'polysub'],
+    'fname': ['polyadd', 'polysub', 'polymul'],
 }))
-class TestPolyArithmeticDiffTypes(unittest.TestCase):
+class TestPolyArithmetic(unittest.TestCase):
```

Comment:
Please restore the name to `TestPoly1dArithmeticDiffTypes` and split the multi-dimension tests.